devicedb/deb/debian/devicedb.service

```diff
@@ -1,7 +1,11 @@
 [Unit]
 Description=DeviceDB: Remote database manager for Maestro
-Requires=edge-proxy.service
-After=maestro.service edge-proxy.service
+PartOf=pelion.service
```

Comment:
I don't think you want `PartOf`` here.  It's only [a one-way dependency](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#PartOf=) so if, for example, edge-core fails, pelion won't show failed.

Reply:
The idea comes from here: https://alesnosek.com/blog/2016/12/04/controlling-a-multi-service-application-with-systemd/.  Any idea how to create a two way dependency?

I think the idea is that edge-core has its own Restart logic, so it shouldn't ever be down.  However, it would be nice to be able to quickly see if edge-core was manually stopped or disabled.

Reply:
Agreed.  Re: two-way... there's probably [something analogous to `PartOf`](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#%5BUnit%5D%20Section%20Options), I'm just not sure what it is.